the fix does make the integration-tests/hibernate-orm-panache module fail with exception:
```
> ./mvnw -am clean install -pl integration-tests/hibernate-orm-panache
...
ERROR] Tests run: 3, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 10.287 s <<< FAILURE! - in io.quarkus.it.panache.PanacheFunctionalityTest
[ERROR] testPanacheInTest  Time elapsed: 0.021 s  <<< ERROR!
javax.enterprise.context.ContextNotActiveException: interface javax.enterprise.context.RequestScoped
        at io.quarkus.hibernate.orm.runtime.RequestScopedEntityManagerHolder_ClientProxy.arc$delegate(RequestScopedEntityManagerHolder_ClientProxy.zig:83)
        at io.quarkus.hibernate.orm.runtime.RequestScopedEntityManagerHolder_ClientProxy.getOrCreateEntityManager(RequestScopedEntityManagerHolder_ClientProxy.zig:191)
        at io.quarkus.hibernate.orm.runtime.entitymanager.TransactionScopedEntityManager.getEntityManager(TransactionScopedEntityManager.java:77)
        at io.quarkus.hibernate.orm.runtime.entitymanager.TransactionScopedEntityManager.createQuery(TransactionScopedEntityManager.java:292)
        at io.quarkus.hibernate.orm.runtime.entitymanager.ForwardingEntityManager.createQuery(ForwardingEntityManager.java:142)
        at io.quarkus.hibernate.orm.panache.runtime.JpaOperations.count(JpaOperations.java:315)
        at io.quarkus.it.panache.Person.count(Person.java)
        at io.quarkus.it.panache.PanacheFunctionalityTest.testPanacheInTest(PanacheFunctionalityTest.java:45) 
```

If I revert the fix, the test succeeds again.
so may be the issue is deeper than just calling the right method?
@gsmet it would worth getting somebody to look at it

--- a/test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestExtension.java
+++ b/test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestExtension.java
@@ -299,13 +299,13 @@ public class QuarkusTestExtension
     @Override
     public void afterEach(ExtensionContext context) throws Exception {
         restAssuredURLManager.clearURL();
-        TestScopeManager.setup();
+        TestScopeManager.tearDown();
     }
 
     @Override
     public void beforeEach(ExtensionContext context) throws Exception {
         restAssuredURLManager.setURL();
-        TestScopeManager.tearDown();
+        TestScopeManager.setup();
     }
 
     @Override

